### PR TITLE
add annotations to std.outbuffer

### PR DESCRIPTION
**Pull request by WalterBright**
Pretty trivial.

### CODE REVIEW

File: std/outbuffer.d [hunk 1]
```diff
@@ -102,12 +99,12 @@ class OutBuffer
             offset += bytes.length;
         }
 
-    void write(in wchar[] chars)
+    void write(in wchar[] chars) @trusted
         {
```
Comment (quickfur) [inline std/outbuffer.d]: Fair enough.

**Comment by quickfur** [comment]
Auto-merge toggled on

**Comment by quickfur** [comment]
Auto-merge toggled off

**Comment by quickfur** [comment]
Actually, looks like the autotester is failing because some dependent functions are not nothrow.

**Comment by braddr** [comment]
own commit, yes.  own pull? not required.

**Comment by andralex** [comment]
Auto-merge toggled on

**Pull request merged**